mPemba/to-do-list
# First Steps

## What was changed
- Adding Firestore DB and Node server. Connected and retrieving data
- Basic UI in place, just to display data for now

## How to test
- Add your Firestore private key config file to the /secrets folder
- Navigate to root and start the server
- `cd server`
- `yarn install`
- `yarn start`
- You should see the message in your terminal "Server is listening on port 5000"

> Now start the front end in a new terminal window
- Navigate to root
- `yarn install`
- `yarn start`

## Screenshots

![Screen Shot 2023-06-03 at 2 30 58 PM](https://github.com/mPemba/to-do-list/assets/8922510/eec991c3-4a90-44fe-adeb-c0967b020f09)
